latch_cli/services/register/models.py

```diff
@@ -187,31 +188,27 @@ def fake_import(name, globals=None, locals=None, fromlist=(), level=0):
         if remote is True:
             headers = {"Authorization": f"Bearer {self.token}"}
 
-            try:
-                user_agent = paramiko.Agent()
-            except paramiko.SSHException as e:
-                raise ValueError(
-                    "An error occurred during SSH protocol negotiation. "
-                    "This is due to a problem with your system. "
-                    "Please use `latch register` without the `--remote` flag."
-                ) from e
-            keys = user_agent.get_keys()
-            if len(keys) == 0:
-                raise ValueError(
-                    "It seems you don't have any (valid) SSH keys set up. "
-                    "Check that any keys that you have are valid, or use "
-                    "a utility like `ssh-keygen` to set one up and try again."
-                )
+            self.ssh_key_path = Path(self.pkg_root) / "ssh_key"
 
-            pub_key = keys[0]
-            alg = pub_key.get_name()
-            material = pub_key.get_base64()
+            # generate private key
```

Comment:
you should check that the ssh-agent binaries are all there before all of this and add nice instructions to download if not